### PR TITLE
Update BaldurKarlsson.RenderDoc to 1.18.0

### PR DESCRIPTION
**Pull request by Kissaki**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59704)

### CODE REVIEW

File: manifests/b/BaldurKarlsson/RenderDoc/1.18.0/BaldurKarlsson.RenderDoc.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  Scope: machine
+  InstallerUrl: https://renderdoc.org/stable/1.18/RenderDoc_1.18_64.msi
+  InstallerSha256: AD10D5E7FDBCC18A36F67C74C0AAB8D8E39A5A463C13B3E2EECD0FA3473BBA4C
+  InstallerSwitches:
+    Custom: /norestart
+  UpgradeBehavior: install
+  ProductCode: '{933D757B-6164-4B2D-9755-E2F23510FA5D}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
```
Comment (Trenly) [inline manifests/b/BaldurKarlsson/RenderDoc/1.18.0/BaldurKarlsson.RenderDoc.installer.yaml:24]:
```suggestion
  InstallerType: wix
```

File: manifests/b/BaldurKarlsson/RenderDoc/1.18.0/BaldurKarlsson.RenderDoc.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+PackageVersion: 1.18.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
```
Comment (Trenly) [inline manifests/b/BaldurKarlsson/RenderDoc/1.18.0/BaldurKarlsson.RenderDoc.installer.yaml:14]:
```suggestion
  InstallerType: wix
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Installer was built with Windows Installer XML Toolset 
```
Name                           Value
----                           -----
Name                           RenderDoc_1.18_64.msi
Size                           74.0 MB
ItemType                       Windows Installer Package
Authors                        Baldur Karlsson
Title                          Installation Database
Subject                        Installer for RenderDoc
Comments                       RenderDoc Homepage is http://renderdoc.org/
ProgramName                    Windows Installer XML Toolset (3.11.1.2318)
FileExtension                  .msi
Filename                       RenderDoc_1.18_64.msi
FolderPath                     D:\Users\Trenly\Downloads\Temp
Path                           D:\Users\Trenly\Downloads\Temp\RenderDoc_1.18_64.msi
```

**Comment by Kissaki** [comment]
Looks like that’s the case for the previous version too, and wrong in the 1.17.0 manifest?

https://renderdoc.org/stable/1.17/RenderDoc_1.17_64.msi

https://github.com/microsoft/winget-pkgs/blob/master/manifests/b/BaldurKarlsson/RenderDoc/1.17.0/BaldurKarlsson.RenderDoc.installer.yaml#L14

**Comment by Kissaki** [comment]
Successfully tested winget validate and install again.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**